### PR TITLE
Add cypress integration test for projects info page

### LINKED ISSUE #94: Write cypress test for projects info page
```
Feature: Project's info page

  Scenario: Access a Project's info page
    Given I am on the Projects Page
    When I click on Anvil project
    Then I should be on the Anvil Project's info page
    And I should see see the tracker link, the github repositories links, a small description, and the number of github commits
```

### PR DESCRIPTION
**Pull request by mattwr18**
Couldn't get this to work with stubbing/fixtures... it kept throwing an error that the request was never sent... found several issues with people using `fetch`, but we are using axios... so I'm not sure what is happening... this is working without the stubbing, but not sure how useful it is since it is actually hitting the api... 
#### Issue addressed

fixes #94

### CODE REVIEW

**Comment by mattwr18** [comment]
and this is failing the build, I've tried increasing the timeout, but it's not finding the element like it is locally @joaopapereira

**Comment by mattwr18** [comment]
got this test passing with stubbing/using fixtures... can you check it out and see what you think @joaopapereira?

File: cypress/integration/common/web_steps.js [hunk 1]
```diff
@@ -1,13 +1,65 @@
...
+    cy.visit('/projects')
+    cy.window()
+      .its('store')
+      .invoke('dispatch', { type: 'GET_PROJECTS', payload: projects })
+  })
+  cy.wait('@getProjects')
+})
+
+When('I click on the LocalSupport project', () => {
+  cy.server()
```
Comment (joaopapereira) [inline cypress/integration/common/web_steps.js]: I would prefer this to be in a Background scenario

File: cypress/integration/projectsInfo.feature [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Feature: Project's info page
+
+  Scenario: Access a Project's info page
+    Given I am at the projects page
```
Comment (joaopapereira) [inline cypress/integration/projectsInfo.feature]: Can we do this in the other test?

Comment (joaopapereira) [reply cypress/integration/projectsInfo.feature]: By this I mean this full test

**Review by joaopapereira — CHANGES_REQUESTED** [review]
I am requesting some changes because I would like to see for now 1 single test that would do:

1. Access the website
1. Click the users menu 
1. checks that the users look fine
1. click on a user
1. check the information
1. Click on the projects menu
1. check that the projects show up correctly
1. click on a project
1. check that the information is present

And so on

File: cypress/integration/common/web_steps.js (hunk 1 above)
Comment (mattwr18) [reply cypress/integration/common/web_steps.js]: when you say this, do you mean `cy.server()`... and by Background scenario you mean like the cucumber background where we would say something like `Given the server has started`?

**Review by joaopapereira — APPROVED** [review]
I think we might need to polish this a little bit more in the future but for now it feels like a good start

**Pull request merged**